examples/create-react-app/WORKSPACE

```diff
@@ -20,6 +17,7 @@ load("@build_bazel_rules_nodejs//:index.bzl", "yarn_install")
 yarn_install(
     # Name this npm so that Bazel Label references look like @npm//package
     name = "npm",
+    data = ["//:patches/jest-haste-map+26.6.2.patch"],
```

Comment:
oops, we should have updated jest so this patch isn't needed, I'll TAL

Reply:
yea. will do in a follow-up across the repo